modules/apps/portal-remote/portal-remote-json-web-service-web/src/test/java/com/liferay/portal/remote/json/web/service/web/internal/JSONWebServiceServiceActionTest.java

```diff
@@ -332,6 +323,24 @@ public boolean isInMemory() {
 			null);
 	}
 
+	private UploadServletRequest _createUploadServletRequest(
```

Comment:
use Proxy in this case instead of using UploadTestUtil.createUploadServletRequest() because later many methods of UploadServletRequest/httpServletRequest will be called during the test. We will need to either mock them one by one, or use Proxy to invoke them. IMO, using proxy will be a cleaner and easier method, and I don't think it is worth to modify UploadTestUtil.createUploadServletRequest() for only this unit test.